test/size.txt

```diff
@@ -1,61 +1,61 @@
       max   |         min   |        gzip   |   file
       ---   |         ---   |         ---   |   ---
  72.23 kB   |     5.87 kB   |     2.53 kB   |   alp.js / alp.max.js
-817.55 kB   |   155.49 kB   |    48.72 kB   |   shadow-v0.js / amp-shadow.js
+835.71 kB   |   161.92 kB   |    51.16 kB   |   shadow-v0.js / amp-shadow.js
    5.8 kB   |       428 B   |       278 B   |   sw-kill.js / sw-kill.max.js
-337.38 kB   |       681 B   |       450 B   |   sw.js / sw.max.js
-852.21 kB   |   156.78 kB   |    49.21 kB   |   v0.js / amp.js
-293.45 kB   |    37.28 kB   |    12.58 kB   |   v0/amp-access-0.1.js
+355.54 kB   |       681 B   |       450 B   |   sw.js / sw.max.js
+870.37 kB   |   163.22 kB   |    51.66 kB   |   v0.js / amp.js
```

Comment:
it gives us custom elements v1 correct as well as the install function dima needs to do shadow dom polyfilling but we can also do the latter with the old version by wrapping the script with a function ourselves.